staging/src/k8s.io/csi-translation-lib/plugins/azure_file_test.go

```diff
@@ -367,9 +368,10 @@ func TestTranslateCSIPVToInTree(t *testing.T) {
 				Spec: corev1.PersistentVolumeSpec{
 					PersistentVolumeSource: corev1.PersistentVolumeSource{
 						AzureFile: &corev1.AzureFilePersistentVolumeSource{
-							SecretName: "azure-storage-account-st-secret",
-							ShareName:  "pvc-file-dynamic",
-							ReadOnly:   true,
+							SecretName:      "azure-storage-account-st-secret",
```

Comment:
Can you also add unit tests for when non default secret namespace is used?

Reply:
added unit tests

Reply:
@msau42 could you take a look again? thanks.

Reply:
Thanks, could you also add unit tests where the share name gets parsed from the VolumeHandle?

Reply:
done, added a new ut